modules/storage-persistent-storage-nfs-group-ids.adoc

```diff
@@ -20,7 +20,7 @@ To gain access to persistent storage, it is generally preferable to use suppleme
 
 Because the group ID on the example target NFS directory
 is `5555`, the Pod can define that group ID using `supplementalGroups`
-under the Pod's `securityContext` definition. For example:
+under the pod's `securityContext` definition. For example:
```

Comment:
s/under the pod's `securityContext` definition./under the `securityContext` definition of the pod.